### ISSUE TITLE
Accessibility : folds should be identified by sound, similar to how breakpoints identified

### ISSUE THREAD

**Issue by RichCaloggero** [microsoft/vscode#169548]
One of the use cases for me when considering vsCode is to have a folding editor. This makes reading someone else's code far easier when using a screen reader.

However, it is currently not possible to identify a fold when moving through the file. I can initiate folding and when viewing my own code, I obviously know that a function that looks empty is "folded".

SUggest playing a cound when entering (and possibly when exiting) a folded region when accessibility mode is enabled. I noticed that breakpoints play a sound when the line containing them is reached; similar thing could be done for folds.

Thanx.
-- Rich

**Comment by RichCaloggero** [comment]
Interestingly, I opened an old js file I had around and I got a couple of different sonic icons.

- when I folded a function, it would tell me it was folded
- when I unfolded it, it would make a sound as I executed the unfold
- when I moved my cursor on a line with an object assignment ( object.foo = "something" ) it made a sound

THis was an old web project, so maybe things are different when operating on web projects; the previous comment was left in response to my examining a node.js project. I don't know why there should be such a difference; they are both clearly javascript...

How can I find a list of all earcons and their meanings?

**Comment by ArturoDent** [comment]
Search for `audio cues` in the Settings to see the available cues.

There is 

```
Audio Cues: Line has Folded Area
Play a sound when the active line has a folded area that can be unfolded.
```

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
This feature request has not yet received the 20 community [upvotes](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) it takes to make to our backlog. 10 days to go. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!
